core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/stack/StackRequestValidator.java

```diff
@@ -139,16 +139,16 @@ private void validateEncryptionKey(StackRequest stackRequest, ValidationResultBu
                 return valueForTypeKey != null && EncryptionType.CUSTOM.equals(getEncryptionIfTypeMatches(valueForTypeKey));
             })
             .forEach(request -> checkEncryptionKeyValidityForInstanceGroupWhenKeysAreListable(request, stackRequest.getCredentialName(),
-                    stackRequest.getAccount(), stackRequest.getRegion(), validationBuilder));
+                    stackRequest.getRegion(), validationBuilder));
     }
 
     private EncryptionType getEncryptionIfTypeMatches(Object o) {
         return o instanceof EncryptionType ? (EncryptionType) o : null;
     }
 
-    private void checkEncryptionKeyValidityForInstanceGroupWhenKeysAreListable(InstanceGroupRequest instanceGroupRequest, String credentialName, String account,
+    private void checkEncryptionKeyValidityForInstanceGroupWhenKeysAreListable(InstanceGroupRequest instanceGroupRequest, String credentialName,
                     String region, ValidationResultBuilder validationBuilder) {
-        Credential cred = credentialService.get(credentialName, account);
+        Credential cred = credentialService.getByNameForOrganization(credentialName);
```

Comment:
Using the method that is lying with its name. The proper ```getByNameForOrganization``` should be used and orgId should be passed. It will be possible once this is merged
https://github.com/hortonworks/cloudbreak/pull/3849

We don't have to wait for it with this PR, its just note.